packages/mock-account-service-lib/src/AccountProvider.ts

```diff
@@ -0,0 +1,271 @@
+interface Account {
```

Comment:
nitpick, but should we lower pascal/kebab case these file names?

Reply:
No thats totally fare. Such as `accountProvider` or `account-provider`? Was accustomed to this way for classes but I see its not our pattern. Looks like we are kinda inconsistent about `accountProvider` vs `account-provider`... renamed this to `accountProvider` (and similar in the `integration` package).

Reply:
we are pretty inconsistent, so `accountProvider` is fine for now

Reply:
i dont really have a strong preference for whatever its worth. This is kinda more what i am used to https://github.com/airbnb/javascript#naming--filename-matches-export but see snake/kebab is recommended by google's style guide https://google.github.io/styleguide/jsguide.html#file-name. I wouldn't be against some filename cleanup.